site/docs/faq.md

```diff
@@ -1,4 +1,50 @@
 ---
 title: FAQ
-sidebar_position: 6
----
\ No newline at end of file
+sidebar_position: 15
+---
+
+## Common questions about using Preevy
+
+<details>
+  <summary>What is the difference between the Livecycle Docker Extension and Ngrok?</summary>
```

Comment:
This is right under "Common question about using *Preevy*" but doesn't relate to Preevy at all.

Maybe have a separate section about the extension?

Reply:
Agreed. We need to update the FAQ content with more meaningful content This is just a start/placeholder.

Reply:
OK, so how about we separate the FAQs of Preevy, extension and Livecycle site to different docs, with the first being a placeholder?

Reply:
zevi